docs/install/network_options.md

```diff
@@ -58,3 +58,92 @@ cp rke2-cilium.yml /var/lib/rancher/rke2/server/manifests/
 ```
 
 For more information about the full options of the Cilium config please refer to the [rke2-charts](https://github.com/rancher/rke2-charts/blob/main-source/packages/rke2-cilium/charts/values.yaml).
+
+# Using Multus
+
+Warning: this is for advanced users
+
+[Multus CNI](https://github.com/k8snetworkplumbingwg/multus-cni) is a plugin that enables attaching multiple network interfaces to pods. Multus does not replace CNI plugins, instead it acts as a CNI plugin multiplexer. Multus is useful in certain use cases, especially when pods are network intensive and require extra network interfaces that support dataplane acceleration techniques such as SRIOV.
+
+Multus should be deployed after the cluster is already up and running and has a CNI plugin taking care of the cluster networking, e.g. Canal. After installing Multus, the current CNI plugin becomes the main CNI plugin under Multus transparently and thus, the cluster can be operated as before, as all pods will get an interface from the main plugin automatically. For more information refer to the [multus-cni](https://github.com/k8snetworkplumbingwg/multus-cni/tree/master/docs)
+
+To deploy Multus, use following config:
+
+```
+apiVersion: helm.cattle.io/v1
+kind: HelmChart
+metadata:
+  name: rke2-multus
+  namespace: kube-system
+spec:
+  repo: https://rke2-charts.rancher.io
```

Comment:
I don't think we want to instruct users to add and pull from this repo directly - its currently just used as a staging point for the RKE2 build pipeline and we need to be careful if we're going to change that.

I think we should package the rke2-multus chart and have users enable it with a CLI flag as discussed at https://github.com/rancher/rke2/issues/746#issuecomment-832541123

Reply:
If user wants multus with default parameters, then he/she adds multus in the cni flag. 
If user wants multus without default parameters, how can he/she change them without pulling from the repo? Or we don't allow this through rke2 and it needs to happen via Rancher?